dev-scripts/get-dpl-react-assets.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
```

Comment:
I am not really happy about including a Python script here. It adds another language dependency that we have to be aware of. 

It creates a dependency for the development environment as we need to ensure that Python3 is available where the task is executed (now: locally or potentially using Docker). It may also create an additional overhead for maintenance.

It is quite a simple script. I suggest that we implement it in PHP or Bash.

A third option might be to create a dependency on the `gh` CLI tool which [supports downloading of assets](https://cli.github.com/manual/gh_release_download).

Reply:
My thought was that the python script is run on the host - so no docker worries.
And another thought was that everyones OS'es runs python.
I knew I would get comments on this. But honestly I don't see the problem about using python.

Reply:
Done